src/client/sandbox/event/listeners.js

```diff
@@ -68,7 +68,7 @@ export default class Listeners extends EventEmitter {
     }
 
     _createEventHandler () {
-        var listeningCtx = this.listeningCtx;
+        var listeners = this;
 
         return function (e) {
```

Comment:
We can not: https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/event/listeners.js#L85